File: frontend_ad/lib/models/order.dart
class Order{
  String? idThuoc;
  String? tenThuoc;
  int? soLuong;
  int? giaTien;
  int? thanhTien;
  Order({this.idThuoc, this.tenThuoc, this.soLuong, this.thanhTien, this.giaTien});
  Order.fromJson(Map<String, dynamic> json){
    idThuoc = json['idThuoc'];
    soLuong = json['soLuong'];
    thanhTien = json['thanhTien'];
  }
  Map<String, dynamic> toJson(){
    Map<String, dynamic> json= Map<String, dynamic>();
    json['idThuoc'] = idThuoc;
    json['soLuong'] = soLuong;
    json['thanhTien'] = thanhTien;
    return json;
  }
}

File: backend/src/schemas/Order.ts
export class Order{
    idThuoc: String;
    soLuong: number;
    thanhTien: number;
    constructor(idThuoc: String, soLuong: number, thanhTien: number){
        this.idThuoc = idThuoc;
        this.soLuong = soLuong;
        this.thanhTien = thanhTien;
    }
}